conda-recipe-generic/build.sh

```diff
@@ -0,0 +1,3 @@
+$PYTHON setup.py build_ext install -- \
+        -DNumPy_INCLUDE_DIR=${SP_DIR}/numpy/core/include \
```

Comment:
I just filed https://github.com/scikit-build/scikit-build/issues/524 because I think scikit-build should find the venv numpy before system numpy.

Reply:
This recipe is taken from the conda-forge feedstock.

Reply:
Are we sure the problem is not the fiddling with the python path in  https://github.com/python-control/Slycot/blob/055e9d62c28801467abf896e307671ea71b78088/CMakeLists.txt#L21-L32 ? 

`find_package(NumPy )` is not a scikit-build feature but from CMake.

Reply:
> find_package(NumPy ) is not a scikit-build feature but from CMake.

I am wrong here. Scikit-build's FindNumpy is a different thing than [FindPython with NumPy component from CMake](https://cmake.org/cmake/help/latest/module/FindPython.html).

Reply:
Cool - that `find_package` finds the right thing without extra magic.  I've noted that on the scikit-build issue.

Reply:
It might not help us (see my comment linking to the disussion there), because it sets different output variables in cmake. We need to find the correct f2py too. I was experimenting with FindF2PY from scikit-build yesterday. Curious if all the old windows hacks in setup.py and the CMake files are still necessary. They do not seem to work for my setup in the workflow so far.